listenbrainz/webserver/views/playlist_api.py

```diff
@@ -691,3 +691,47 @@ def copy_playlist(playlist_mbid):
         raise APIInternalServerError("Failed to copy the playlist. Please try again.")
 
     return jsonify({'status': 'ok', 'playlist_mbid': new_playlist.mbid})
+
+
+@playlist_api_bp.route("/<playlist_mbid>/export/<service>", methods=["POST", "OPTIONS"])
+@crossdomain
+@ratelimit()
+@api_listenstore_needed
+def export_playlist(playlist_mbid, service):
+    """
+
+    Export a playlist to an external service.
+
+    :reqheader Authorization: Token <user token>
+    :statuscode 200: playlist copied.
+    :statuscode 401: invalid authorization. See error message for details.
+    :statuscode 404: Playlist not found
+    :resheader Content-Type: *application/json*
+    """
+    user = validate_auth_header()
+
+    if not is_valid_uuid(playlist_mbid):
+        log_raise_400("Provided playlist ID is invalid.")
+
+    if service != "spotify":
+        raise APIBadRequest(f"Service name {service} is invalid or currently not supported")
```

Comment:
"Service {service} is not supported. We currently only support 'spotify'."

I love it when error messages answer any possible questions I have, like "what services are provided".